rts/lra-examples/run.sh

```diff
@@ -47,14 +47,6 @@ function killpid {
   test $? || echo "===== could not kill $1"
 }
 
-function start_coordinator {
```

Comment:
@jmfinelli why have you removed this function for starting the coordinator.
The purpose of a quickstart is help the user become familiar with how to use the s/w - now it's less clear how to start coordinators. The purpose of functions is to create modular readable software, can you re-instate it please.

Reply:
I removed this function as it was not used. The same code was already pasted in the `if` at (the old) code line 125. If you think we should keep it anyway, I'll re-instate it

Reply:
If it's not used then you were right to remove it, sorry.